java/distance-measurement-analysis/README.md

```diff
@@ -1,33 +1,33 @@
-# Distance Measurement Analysis
-Measure 3D distances between two points in a scene.
+# Distance measurement analysis
 
-The distance measurement analysis allows you to add the same measuring experience found in ArcGIS Pro, City Engine,
-and the ArcGIS API for JavaScript to your app. You can set the unit system of measurement (metric or imperial) and 
-have the units automatically switch to one appropriate for the current scale. The rendering is handled internally 
-so it doesn't interfere with other analyses like viewsheds.
+Measure distances between two points in 3D.
 
-![Distance Measurement Analysis App](distance-measurement-analysis.png)
+![Image of distance measurement analysis](distance-measurement-analysis.png)
+
+## Use case
+
+The distance measurement analysis allows you to add to your app the same interactive measuring experience found in ArcGIS Pro, City Engine, and the ArcGIS API for JavaScript. You can set the unit system of measurement (metric or imperial). The units automatically switch to one appropriate for the current scale.
 
 ## How To Use The Sample
-Choose an unit system for the measurement in the UI dropdown. Single tap any location on the screen to set the start point
-of measurement. Double tap and drag to set the end point of measurement. Single Tapping any new location will 
-start a new measurement. 
+Choose an unit system for the measurement in the UI dropdown. Single tap any location on the screen to set the start point of measurement. Double tap and drag to set the end point of measurement. Single Tapping any new location will start a new measurement.
+
+## How it works
 
-## How It Works
-To measure distances with the `LocationDistanceMeasurement` analysis:
+1. Create an `AnalysisOverlay` object and add it to the analysis overlay collection of the `SceneView` object.
+2. Specify the start location and end location to create a `LocationDistanceMeasurement` object. Initially, the start and end locations can be the same point.
+3. Add the location distance measurement analysis to the analysis overlay.
+4. The `measurementChanged` callback will trigger if the distances change. You can get the new values for the `directDistance`, `horizontalDistance`, and `verticalDistance` from the `MeasurementChangedEvent` object returned by the callback.
```

Comment:
let's refer to the returned callback as lower case to make it clear it's not a class in our API
```suggestion
4. The `measurementChanged` callback will trigger if the distances change. You can get the new values for the `directDistance`, `horizontalDistance`, and `verticalDistance` from the `measurementChangedEvent` object returned by the callback.
```